http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -231,6 +232,7 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                   storageContainer,
                   BatchAccountKey(batchAccountKey),
                   applicationInsightsComponent.connectionString(),
+                  params.userAccessToken,
```

Comment:
Since this is a temporary solution, I think it would be nice to put a TODO before this line (as well as in the WDS case below to warn future devs.

TODO: Remove once permanent solution utilizing the multi-user sam app identity has been implemented